FlubberView/FlubberView.swift

```diff
@@ -126,10 +129,18 @@ public extension FlubberView {
             mainAnimator.addBehavior(bounceBehavior)
             v.center = CGPoint(x: v.center.x <~> magnitude.elasticity, y: v.center.y <~> magnitude.elasticity)
             mainAnimator.updateItem(usingCurrentState: v)
+
+            // Debounce snap behavior
+            if let snap = snapBehavior {
+                mainAnimator.removeBehavior(snap)
+            }
+
+            // Snap back to original position
             DispatchQueue.main.asyncAfter(deadline: DispatchTime.now() + duration, execute: {
                 let snap = UISnapBehavior(item: v, snapTo: initialPoint)
```

Comment:
#req I started teasing this apart, and the snap behavior is being set for every point, and the operation still fires every time, the snap behaviors need to be stored in a map table like the attachment behaviors, and then the update needs to be scheduled at the end of the loop for every snap operation inside an Operation that is canceled if the view is clicked again.